src/matrix/inversewishart.jl

```diff
@@ -43,6 +43,16 @@ size(d::InverseWishart) = (p = dim(d); (p, p))
 params(d::InverseWishart) = (d.df, d.Ψ, d.c0)
 @inline partype{T<:Real}(d::InverseWishart{T}) = T
 
+### Conversion
+function convert{T<:Real}(::Type{InverseWishart{T}}, d::InverseWishart)
+    P = convert_eltype(T, d.Ψ)
```

Comment:
So I tweaked the code. `convert_eltype` still seems to me the cleanest solution for now, but it's essentially just aliasing `convert` methods of the type you suggested, so it should be easy to swap out for an alternative in Base later, if available.